testplan/testing/multitest/base.py

```diff
@@ -61,14 +60,14 @@ def iterable_suites(obj):
     return suites
 
 
-class MultiTestRuntimeInfo(object):
+class MultiTestRuntimeInfo(testing_base.RuntimeInfo):
```

Comment:
To keep it simple I have removed the implementation. the only change is rename 'multitest_runtime_info' to 'runtime_info' because this member is already in 'MultiTestInfo' class.